portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/upgrades/DatabaseUpgradeClient.testcase

```diff
@@ -59,4 +59,12 @@ definition {
 			value1 = "check-upgrade-client-zip-content");
 	}
 
+	@description = "LPS-168551: Given portal-ext.properties in the user's home directory. When the user uses the DB Upgrade Tool Then the Upgrade tool returns 'Remove /home/${USER}/portal-ext.properties prior to running an upgrade to prevent possible conflicts.' error"
```

Comment:
I think we can simplify the description. Maybe to something like: "LPS-168551: Upgrade client throws error when portal-ext.properties exists in home directory"